---
title: "Miao Lab - Research"
layout: textlay
excerpt: "Miao Lab -- Research"
sitemap: false
permalink: /research/
---

# Research

Our lab will mainly use computational approaches combined with experiments to understand RNA virus and RNA structure at gene expression regulation and structural levels.
Specifically, we use RNA structure prediction and RNA design to udnerstand and engineer RNA molecules. We are hosting the [RNA-Puzzles](http://www.rnapuzzles.org), which is a world-wide community efforts in evaluation of RNA 3D structure prediction. We explore RNA structural biology using a combination of X-ray crystallography, chemical probing, artificial intelligence and computational modeling. 
Besides, we use single-cell multi-omics to understand the human immune response to RNA viruses, neuron development and immune cell development. We develop computational algorithms to deal with key bottlenecks in single-cell analysis (e.g.,batch effect, cell type clustering, cell type deconvolution). We also develop cloud computing platform and big data atlases for single-cell omics.

Here are some themes that we currently work on:

**[RNA-Puzzles.](http://www.rnapuzzles.org)** It is a collective experiment for blind RNA structure prediction.    

It aims to:     
* To determine the capabilities and limitations of current methods of 3D RNA structure prediction based on sequence;    
* To find whether and how progress has been made, as well as what has yet to be done to achieve better solutions    
* To identify whether there are specific bottlenecks that hold back the field;
* To promote the available methods and guide potential users in the choice of suitable tools for real-world problems;
* To encourage the RNA structure prediction community in their efforts to improve the current tools and to make automated prediction tools available.
* To explore the underlying mechanism of ligand-RNA binding and the conformational changes in riboswitches.

For details, please visit [rnapuzzles.org](http://www.rnapuzzles.org).
<img src="{{ site.url }}{{ site.baseurl }}/images/homepic/rnapuzzles_screenshot.png" alt="Slide 1" style="height:400px;"/>

**RNA design.** We aim to design catalytic RNA molecule as well as RNA inhibitors using the RNA structure folding rules. 

It is widely known that RNA form base-pairs and secondary structures. However, the "base-pairs" are normally referring to *cis*-Watson-Crick base-pairs of A-U or G=C, sometimes G-U, while 
the non-Watson-Crick base-pairs also play a key role in RNA structure formation. We aim to integrate experimental information and RNA structural rules derived from X-ray crystal structures 
to optimise RNA design including the non-Watson-Crick base-pairs. 

**Single-cell omics algorithms.** The 


**Single-cell omics data analysis.**

